libs/expo/shared/ui-components/src/lib/Input/Input.tsx

```diff
@@ -149,7 +149,7 @@ export function Input(props: IInputProps) {
                 accessibilityRole="button"
                 accessibilityLabel="delete icon"
                 accessibilityHint="deletes input's value"
-                onPress={() => onChange('')}
+                onPress={() => onChange(null)}
```

Comment:
I'd rather avoid doing this globally as may have unexpected consequences (types aren't checked here and I'm not sure what setting values to `null` on any single field would do).

If the server really can't accept an empty string for `email`, then perhaps may be safer to cast empty string to `null` somewhere in the [AddEditClient/index.tsx](https://github.com/BetterAngelsLA/monorepo/blob/dd7bd55be46db37a6735dba763c5b678a4587d9e/libs/expo/betterangels/src/lib/screens/AddEditClient/index.tsx#L156)

would something like this work?
don't remember if we have other email fields (for other contacts maybe) that  would need a similar thing.

```javascript
    if (typeof values.user?.email === 'string') {
      values.user.email = values.user.email || null;
    }
```

Reply:
that's a good point. I think your idea will work